types/node/index.d.ts

```diff
@@ -756,20 +761,23 @@ declare module "http" {
         write(str: string, encoding?: string, cb?: Function): boolean;
         write(str: string, encoding?: string, fd?: string): boolean;
 
-        writeContinue(): void;
-        writeHead(statusCode: number, reasonPhrase?: string, headers?: any): void;
-        writeHead(statusCode: number, headers?: any): void;
-        statusCode: number;
-        statusMessage: string;
+        addTrailers(headers: ServerResponseHeaders): void;
+        finished: boolean;
+        getHeader(name: string): number | string | string[] | undefined;
```

Comment:
My feeling is that content-type definition should be adapted here as getHeader() may return something else then a string in general so the definition here matches better to the actual code.
For the content-type case getHeader() is always called with 'content-type' which should in real world code be either a string or undefined but not string[],...